### DISCUSSION TITLE
Project and Project - Mapped Field Generation

### DISCUSSION THREAD

**Discussion by shawkatmarrouf** [YetiForceCompany/YetiForceCRM#16270] [How to] [answered]
Yetiforce 6.3 | When creating a project task generated mapped fields from project it doesn't list or relate to that project or appear in summary, I've to create the record manually to list the task inside project tasks tap.

I've tried one to many field and generate it, but what happened is that another Project task tap appeared beside the original one and it lists from generated mapped field, but the original one is empty and not appear in the summary tap, that's confusing!!

Is there any solution to list the project tasks generated from mapped field in its original tap and to display it in the summary tap as it happens when creating the task manually from Project task tap ?  @prabhu @joebordes @vctrin @Raggazzoo

**Comment by apcloic** [comment, marked as answer]
@shawkatmarrouf
I don't understand your problem, can you show us and record a video on gitstable.yetiforce.com using [Screen2gif](https://www.screentogif.com/) ?

**Reply by shawkatmarrouf** [reply]
@apcloic 
In Projects module there is a hard coded "Project Tasks" tap (highlighted with Orange).
In the meantime, our project task fields is the same as project fields, so I've created Mapped Fields between the project and project tasks to save so much time.

after I created mapped fields, another "Project Tasks" tap appeared in Project module.
when generating a project task using the highlighted arrow button "Mapped Fields" I found that the task is being listed only in this new "Project Tasks" tap (highlighted with red), and doesn't appear in the original one "highlighted with Orange".

In the summary view (highlighted with orange) the task listed here is only the task that appears in the original "Project tasks" tap. so that's confusing!

![git01](https://user-images.githubusercontent.com/101041720/179947766-8b46503a-a0be-4bb8-a97f-16885523d9a2.png)

I want to use the original "Project task" tap that is connected with the summary view when generating mapped fields.

Thanks!

**Reply by apcloic** [reply]
@shawkatmarrouf 
It looks like you have 2 different types of relation between Project & Project tasks, you could check that on vtiger_relatedlists table.
There is no reason to have another 'Project tasks' tab on project module, did you test on https://gitstable.yetiforce.com/ ?
You should only have a getDependentsList relation type between project and project task module.
